src/meta-assistant/slackapi/slackapi-utils.ts

```diff
@@ -0,0 +1,243 @@
+import { DailyCombinedData, WeeklyCombinedData, TimeRegistrations, PreviousWorkdayDates, NonProjectTime, DailyMessageData, DailyMessageResult, WeeklyMessageData, WeeklyMessageResult } from "src/types/meta-assistant/index";
+import { ChatPostMessageResponse, LogLevel, WebClient } from "@slack/web-api";
+import { Member } from "@slack/web-api/dist/response/UsersListResponse";
+import { DateTime } from "luxon";
+import TimeUtilities from "../generic/time-utils";
+
+/**
+ * Namespace for Slack API utilities
+ */
+namespace SlackApiUtilities {
+
+  export const client = new WebClient(process.env.METATAVU_BOT_TOKEN, {
+    logLevel: LogLevel.DEBUG
+  });
+
+  const listOfIds = process.env.STAGING_IDS ? process.env.STAGING_IDS.split(",") : undefined;
```

Comment:
Staging ids?

Reply:
Updated to SLACK_USER_OVERRIDE. Previously some envs were not validated, not all are.

Reply:
Also added the schedule cron timer as an env